dts/arm/nordic/nrf5340_cpuappns.dtsi

```diff
@@ -48,6 +48,24 @@
 			 */
 			#include "nrf5340_cpuapp_peripherals.dtsi"
 		};
+
+		/* Cryptocell is a secure-only peripheral, but the availability
+		 * of this peripheral and the cryptocell variant is still useful
+		 * information for the non-secure application as it can access
+		 * crypto features through secure-services.
+		 */
+		cryptocell: crypto@50844000 {
+			compatible = "nordic,nrf-cc312";
+			reg = <0x50844000 0x1000>;
+			status = "okay";
```

Comment:
```suggestion
			status = "disabled";
```

Reply:
peripheral enablement needs to be performed at board level (if needed) or by applications. exceptions are those which are always needed.

Reply:
In the NS application this would build against TF-M, which is using Cryptocell, for RNG and crypto both for itself and as crypto services to the NS application.